tests/clients/layer0_test_client.go

```diff
@@ -60,43 +81,77 @@ func (l *Layer0TestClient) CreateLoadBalancer(name, environmentID string) *model
 
 	ports := []models.Port{{HostPort: 80, ContainerPort: 80, Protocol: "http"}}
 
-	loadBalancer, err := l.Client.CreateLoadBalancer(name, environmentID, hc, ports, true)
+	req := models.CreateLoadBalancerRequest{
+		LoadBalancerName: name,
+		EnvironmentID:    environmentID,
+		IsPublic:         true,
+		Ports:            ports,
+		HealthCheck:      hc,
+	}
+
+	jobID, err := l.Client.CreateLoadBalancer(req)
 	if err != nil {
 		l.T.Fatal(err)
 	}
 
-	return loadBalancer
+	return jobID
 }
 
-func (l *Layer0TestClient) CreateService(name, environmentID, deployID, loadBalancerID string) *models.Service {
-	service, err := l.Client.CreateService(name, environmentID, deployID, loadBalancerID)
+func (l *Layer0TestClient) CreateService(name, environmentID, deployID, loadBalancerID string) string {
+	req := models.CreateServiceRequest{
+		DeployID:       deployID,
+		EnvironmentID:  environmentID,
+		LoadBalancerID: loadBalancerID,
+		ServiceName:    name,
+		Scale:          1,
```

Comment:
[nit] does this have to be set?

Reply:
It looks like the answer is no given that the `aws` package will set this value to 1 if no value is specified in the request.